.changeset/bitter-teeth-think.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudflare/vite-plugin": patch
```

Comment:
I think this should be minor. It's a feature, not a bug fix, right?

Reply:
This is what was agreed in the meeting yesterday for beta and `0.x.x` packages.

> Minor for breaking changes, patch for bug fixes and new backwards-compatible features